amc2moodle/amc2moodle/README.md

```diff
@@ -36,11 +36,27 @@ Examples of the `amc2moodle` possibilities are given at [QCM.pdf](./test/QCM.pdf
   -  Use italic, typerwritter, bold, emphasize...
   -  Use enumerate and itemize (but without the tag `\item[tag]`) 
   -  Automatically add an answer like ``there is no good answer'' if there is no good answer.
+  -  All answers are Shuffled by default, you can keep the answer initial order by setting `ShuffleAll = False` in `grading.py`
```

Comment:
(I'm working on a patch that adds support for `\begin{choices}[o]...\end{choices}` in ams2moodle.  It's nearly done, but I'm waiting for this PR to be merged before I send mine.)